getting-started/windows-calico/kubernetes/standard.md

```diff
@@ -120,9 +130,15 @@ In addition, it's important that `kubelet` is started after the vSwitch has been
 
 - Waits for {{site.prodname}} to initialise the vSwitch
 - Starts `kubelet` with
-    - --network-plugin set to `cni`
-    - --cni-bin-dir set to `c:\k\cni`
-    - --cni-conf-dir set to `c:\k\cni\config`
+    - If containerd service is running, the following flags are set:
+        - --container-runtime set to `remote`
+        - --container-runtime-endpoint set to `npipe:////.//pipe//containerd-containerd`
+    - Otherwise, the following flags are set for Docker:
+        - --network-plugin set to `cni`
+        - --cni-bin-dir set to `c:\k\cni`
+        - --cni-conf-dir set to `c:\k\cni\config`
+        - --pod-infra-container-image set to `kubeletwin/pause`
+        - --image-pull-progress-deadline set to `20m`
```

Comment:
**`- --image-pull-progress-deadline set to `20m`** I don't think this has any significance on how user should set kubelet arguments. Maybe we shouldn't mention it here.